tensordict/base.py

```diff
@@ -2941,16 +2943,38 @@ def map(
                 of workers. For very large tensordicts, such large chunks
                 may not fit in memory for the operation to be done and
                 more chunks may be needed to make the operation practically
-                doable. This argument is exclusive with num_chunks.
+                doable. This argument is exclusive with ``num_chunks``.
             num_chunks (int, optional): the number of chunks to split the tensordict
                 into. If none is provided, the number of chunks will equate the number
                 of workers. For very large tensordicts, such large chunks
                 may not fit in memory for the operation to be done and
                 more chunks may be needed to make the operation practically
-                doable. This argument is exclusive with chunksize.
+                doable. This argument is exclusive with ``chunksize``.
             pool (mp.Pool, optional): a multiprocess Pool instance to use
                 to execute the job. If none is provided, a pool will be created
                 within the ``map`` method.
+            seed (integer, optional): the initial seed of the pool. Each member
```

Comment:
`DataLoader` doesn't take a seed as input but rather[ `Generator` object](https://github.com/pytorch/pytorch/blob/8ec59d3553db4e3dae991db9aa3251558265dfd3/torch/utils/data/dataloader.py#L169C1-L171C59), which defaults to the pytorch default RNG. The `base_seed` is then [drawn internally](https://github.com/pytorch/pytorch/blob/8ec59d3553db4e3dae991db9aa3251558265dfd3/torch/utils/data/dataloader.py#L602C6-L602C6) from that `Generator`

Reply:
thanks I will edit that!